nautobot/dcim/models/device_components.py

```diff
@@ -501,6 +504,10 @@ class Meta:
 
     def save(self, *args, **kwargs):
 
+        # set interface status to active if status not provided
+        if not self.status:
+            self.status = Status.objects.get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
```

Comment:
Maybe something like this would be safer?

```suggestion
        if not self.status:
            interface_content_type = ContentType.objects.get_for_model(Interface)
            try:
                self.status = Status.objects.get(slug=InterfaceStatusChoices.STATUS_ACTIVE, content_types__in=[interface_content_type])
            except Status.DoesNotExist:
                self.status = Status.objects.filter(content_types__in=[interface_content_type]).first()
```